### DISCUSSION TITLE
Toggle hidden git ignore files not working at all

### DISCUSSION THREAD

**Discussion by ChanManChan** [nvim-tree/nvim-tree.lua#2720] [Q&A]
'I' does not toggle git ignore files anymore. They are always visible for some reason. I use the same configs across multiple computers and yet only face this issue on my secondary computer

my nvim tree config::

```
local options = {
  filters = {
    dotfiles = false,
    exclude = { vim.fn.stdpath "config" .. "/lua/custom" },
  },
  disable_netrw = true,
  hijack_netrw = true,
  hijack_cursor = true,
  hijack_unnamed_buffer_when_opening = false,
  sync_root_with_cwd = true,
  update_focused_file = {
    enable = true,
    update_root = false,
  },
  view = {
    adaptive_size = false,
    side = "left",
    width = 30,
    preserve_window_proportions = true,
  },
  git = {
    enable = true,
    ignore = true,
  },
  filesystem_watchers = {
    enable = true,
  },
  actions = {
    open_file = {
      resize_window = true,
    },
  },
  renderer = {
    root_folder_label = false,
    highlight_git = false,
    highlight_opened_files = "none",

    indent_markers = {
      enable = false,
    },

    icons = {
      show = {
        file = true,
        folder = true,
        folder_arrow = true,
        git = true,
      },

      glyphs = {
        default = "󰈚",
        symlink = "",
        folder = {
          default = "",
          empty = "",
          empty_open = "",
          open = "",
          symlink = "",
          symlink_open = "",
          arrow_open = "",
          arrow_closed = "",
        },
        git = {
          unstaged = "✗",
          staged = "✓",
          unmerged = "",
          renamed = "➜",
          untracked = "★",
          deleted = "",
          ignored = "◌",
        },
      },
    },
  },
}

return options
```

when I press shift + i multiple times I get this error at the bottom
![2024-03-21-160718](https://github.com/nvim-tree/nvim-tree.lua/assets/42992965/3c33b5fb-5e1c-493b-81f4-f8043926cbc2)

**Comment by ChanManChan** [comment]
All of a sudden it's working now for some reason!? I don't understand. I have been trying to get this working for some time now. Why is this happening?

**Reply by alex-courtis** [reply]
It looks like git timed out; it's not going to report git ignored.

Try increasing `git.timeout` to 800; that's what I'm using for some massive repos.

**Reply by ChanManChan** [reply]
Yea but my under powered laptop was able to handle this repo on the first try without increasing the timeout. Yes I agree the repo is massive but I got confused as my top of the line desktop was struggling for some reason

**Reply by alex-courtis** [reply]
Git can be unpredictable, with the little maintenance operations it performs from time to time.

Don't be too concerned; git is asynchronous so won't affect nvim responsiveness.
